CMakeLists.txt

```diff
@@ -91,6 +91,10 @@ endif()
 
 find_package(opm-common REQUIRED)
 
+if(USE_MPI)
+  set(HDF5_PREFER_PARALLEL TRUE)
+endif()
```

Comment:
This is probably the right thing to do in general, but CMake has an [issue](https://gitlab.kitware.com/cmake/cmake/-/issues/19117) that's been open for three years stating that `PREFER_PARALLEL` doesn't really do what it's supposed to.

What's your experience?

Reply:
i haven't had any issues with it on debian derivates. that anaconda creates issues is business as usual.

Reply:
> i haven't had any issues with it on debian derivates.

Understood.  Then I'm happy to have this.